workshop2/01-setup.md

```diff
@@ -0,0 +1,83 @@
+⏮ ~~Previous step~~&nbsp;|&nbsp;[🏠 Up](./00-index.md)&nbsp;|&nbsp;[⏭ Next step: Add endpoints](./02-create-endpoints.md)
+
+----
+
+## Step 1: Get your environment set up
+
+In this step, you will be using the repository we've created for you for the [previous workshop](../workshop1), and you'll use [.NET Core CLI] to set up the project for your animal shelter REST API.
+
+### Pre-requisites
+
+For this workshop we are going to use [Visual Studio Code] with the C# extension and [.NET Core CLI] to create the project and add nuget packages to it.
+
+You can download and install everything from these links:
+- [Visual Studio Code]
+- [C# for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp)
+- [.NET Core CLI]
+
+### Cloning your repository
+
+In the previous workshop, you were assigned with a repository where you commited all the UI work. In this workshop, we will continue using that repository to commit the REST API code.
+
+If you can't remember what was the name of your repo, you can go to your github account and in the repositories look for one with a name something like `happy-rover` within the [CodessInTheClassroom GitHub organisation].
+
+If you still have the repo cloned locally, you can continue with the next section. If you delete it, you can clone it again. For this open up your preferred shell:
+
+1. Create a local folder to keep your code and change to it. For example, to
+   create `C:\CodessInTheClassroom`:
+
+       mkdir C:\CodessInTheClassroom
+       cd C:\CodessInTheClassroom
+
+2. Find your repository within the [CodessInTheClassroom GitHub organisation].
+
+3. Clone your repository using the instructions in [our Git
+   cheatsheet](../git-cheatsheet.md#clone). For example:
+
+       git clone https://github.com/codessintheclassroom/happy-rover.git
+
+### Creating a branch for this workshop work
+
+Now we'll create a new branch to hold all the new code for the REST API. This will allow to submit a pull request at the end and get your code review.
+
+1. [Follow the instructions for creating a branch in the Git
+   cheatsheet](../git-cheatsheet.md#create-a-branch). You should give it a
+   descriptive name so that you know what it's for, such as
+   `yourname/shelter-backend` (on some teams developers prefix their branch
+   names with their usernames) or `shelter-backend`.
+
+### Creating the initial application code
+
+Now we'll use [.NET Core CLI] to create the initial code for our
+application:
+
+1. Using your favourite shell, go to the cloned repo location, for example, if you repository was called `happy-rover`, run `cd c:\CodessInTheClassroom\happy-rover` and run the following command to  generate a web api project `dotnet new webapi -n backend`. This will create a new C# project with some example code for a weather forecast api, that we will override to create our animal shelter API.

```

Comment:
```suggestion
1. Using your favourite shell, go to the cloned repo location (for example, if your repository was called `happy-rover`, run `cd c:\CodessInTheClassroom\happy-rover`) and run the following command to  generate a web API project: `dotnet new webapi -n backend`. This will create a new C# project with some example code for a weather forecast api, that we will replace to create our animal shelter API.
```